src/tier0/threadtools.cpp

```diff
@@ -267,6 +267,11 @@ unsigned int g_optimalMaxNormalizedYieldsPerSpinIteration = 7;
 const unsigned int MinNsPerNormalizedYield = 37; // measured typically 37-46 on post-Skylake
 const unsigned int NsPerOptimalMaxSpinIterationDuration = 272; // approx. 900 cycles, measured 281 on pre-Skylake, 263 on post-Skylake
 
+#ifdef POSIX
+bool s_isYieldProcessorNormalizedInitialized = false;
```

Comment:
I would rather you do an implementation of Init shich just sets s_isYieldProcessorNormalizedInitialized to true, and add a todo.

Reply:
I meant to set it to true, will change as requested.